src/spotishush.js

```diff
@@ -26,6 +26,12 @@ const SITES = {
     name: 'IDAGIO',
     log_css: 'background: #fff; color: #000; font-weight: bold',
     init: idagioInit
+  },
+  
+  'music.youtube.com': {
+      name: 'Youtube',
+      log_css: 'background: #fff; color: #000; font-weight: bold',
```

Comment:
I'm thinking we can maybe change the background color to `#000` and the text color to a red-ish one (your choice).